CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 10.0.2 (YYYY-MM-DD)
+
+### Breaking Changes
+* None.
+
+### Enhancements
+* None.
+
+### Fixes
+* [RealmMigration] Fixed crash when adding classes containing an `ObjectId` as primary key to the schema (Issue [#7189](https://github.com/realm/realm-java/issues/7189), since v10.0.0).
```

Comment:
```suggestion
* Fixed crash when adding classes containing an `ObjectId` as primary key to the schema (Issue [#7189](https://github.com/realm/realm-java/issues/7189), since v10.0.0).
```

We only use the `[RealmApp]` suffix when a change is for the sync API's. Otherwise we don't attach a "group"